### PR TITLE
(chore) Prepare for 2.6.0 release

### PR DESCRIPTION
**Pull request by Mat001**
## Summary
- Update changelog for a new minor release 2.6.0 - forced decisions

## Issues
- https://optimizely.atlassian.net/browse/OASIS-7947

### CODE REVIEW

**Comment by msohailhussain** [comment]
Please add proper description in the PR.

**Comment by msohailhussain** [comment]
Can you please add the openapi-spec link in this PR or description that explains how to use forced decision.

**Review by jaeopt — CHANGES_REQUESTED** [review]
A few clean ups suggested

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.6.0] - Jan 13, 2022
+
+- Introduce `Forced Decisions` property into the `decide` API for overriding and managing user-level flag, experiment and delivery rule decisions. Forced decisions can be used for QA and automated testing purposes ([#324](https://github.com/optimizely/agent/pull/324), [#325](https://github.com/optimizely/agent/pull/325)).
+
+    - For details about this Agent release, refer to our API documentation page: https://library.optimizely.com/docs/api/agent/v1/index.html#operation/decide.
```
Comment (jaeopt) [inline CHANGELOG.md:11]:
```suggestion
    - For details, refer to our API documentation page: https://library.optimizely.com/docs/api/agent/v1/index.html#operation/decide.  
```

**Review by jaeopt — APPROVED** [review]
LGTM - a nit cleanup

**Pull request merged**